lib/ansible/modules/system/iptables.py

```diff
@@ -359,6 +327,13 @@ def append_param(rule, param, flag, is_list):
             rule.extend([flag, param])
 
 
+def append_match_flag(rule, param, flag, negatable):
```

Comment:
So negatable means that the match param actually can be negated, meaning it's an option that makes works when you put a '!' in front of it. If a match option is not negatable, meaning it doesn't work with an exclamation mark it should get False as parameter value. So for syn it's set to True because it can be negated.